### PR TITLE
BAN-88: Add notification component

### PR DESCRIPTION
**Pull request by mihaelpalfi**
https://reciprocitylabs.atlassian.net/browse/BAN-88

Closed previous [PR](https://github.com/reciprocity/zen-ui/pull/37) and recreated it here. Added the review requested changes fixes.

### CODE REVIEW

File: src/components/zen-notification/zen-notification.stories.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import { html } from 'lit-html';
+import markdown from './readme.md';
+import { ZenDismissDuration, ZenVariant } from './zen-notification-helper';
+
+const argTypes = {
```
Comment (f-jauregui) [inline src/components/zen-notification/zen-notification.stories.ts:5]: Arg types that are not type select can be put into a Default.args object, check how zen-step does that, that will inference automatically data type.
Remove description and type from variant and dismissDuration.

File: src/zen-styles/colors.scss [hunk 2]
```diff
@@ -25,6 +25,7 @@ $color-red-100: #fcdadd;
 $color-green-900: #1b422e;
 $color-green-800: #2a6949;
 $color-green-700: #34835b;
+$color-green-600: #3bbf74;
```
Comment (f-jauregui) [inline src/zen-styles/colors.scss:28]: That green doesn't exist in color palette of the design system. Maybe you can use green-500 or green-700 to achieve that. Should be same color as border-color.

**Review by f-jauregui — APPROVED** [review]
Just those little changes.
It's looking better now! 😎

**Comment by f-jauregui** [comment]
Don't forget to create a story to add complete spec tests. Those tests are not enough.

File: src/components/zen-notification/zen-notification.stories.ts (hunk 1 above)
Comment (mihaelpalfi) [reply src/components/zen-notification/zen-notification.stories.ts:5]: Done

File: src/zen-styles/colors.scss (hunk 2 above)
Comment (mihaelpalfi) [reply src/zen-styles/colors.scss]: Done

**Pull request merged**